opentreemap/treemap/templates/instance_base.html

```diff
@@ -134,8 +120,8 @@
   {% endif %}
 
   <div class="stats-bar hidden-xs">
-    <div class="half">
-      <div style="display: inline;" id="tree-and-planting-site-counts">
+    <div class="stats-list">
+      <div id="tree-and-planting-site-counts">
```

Comment:
That is more accurate, but there is JS code that uses this `id`, and I don't think it's a great use of our time to go changing it everywhere it is used.